setup.py

```diff
@@ -85,7 +85,7 @@ def get_package_data(parent, roots):
 
 setup(
     name="pyfarm.agent",
-    version="0.8.3",
+    version="0.8.3.1",
```

Comment:
I'll go ahead and merge this then, I'll roll out 0.8.3 tomorrow unless you'd prefer I'd not.